server/src/main/java/io/crate/role/RoleManagerService.java

```diff
@@ -116,7 +116,11 @@ public CompletableFuture<Long> createRole(String roleName,
                                               @Nullable JwtProperties jwtProperties) {
         return transportCreateRoleAction.execute(new CreateRoleRequest(roleName, isUser, hashedPw, jwtProperties), r -> {
             if (r.doesUserExist()) {
-                throw new RoleAlreadyExistsException(roleName);
+                if (jwtProperties == null) {
+                    throw new RoleAlreadyExistsException(roleName);
+                } else {
+                    throw new RoleAlreadyExistsException(roleName, jwtProperties);
+                }
             }
```

Comment:
Another approach could be introducing `JwtPropertiesAlreadyExists` exception and adding jwtExists to `WriteRoleResponse`.

I felt like having user with conflicting jwt props kinda fits into "role exists" but not sure. wdyt?